scripts/archive_images.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import re
+from pathlib import Path
+
+from common.image_repository_helpers import (
+    DeleteVersionStatus,
+    Image,
+    Repository,
+    add_tags,
+    copy_image,
+    delete_version,
+    list_images_in_repository,
+    version_exists,
+)
+
+logging.getLogger().setLevel(logging.INFO)
+
+SUPPORTED_REPOSITORIES = ['images']
+
+
+def get_archive_set():
+    archive_list_file = Path(__file__).parent.parent / 'archived_images.txt'
+    archive_list: list[str] = []
```

Comment:
It could be a set here too, and add each line with `archive_set.add(…)`, but no biggie.

Reply:
yeah good point, may as well avoid the conversion